lib/data/copy/check_in.yml

```diff
@@ -3,39 +3,57 @@ first_greeting: >
 
     I'll be reaching out to you every week, typically on Fridays, to check in and see how your club's doing. I'll be in the loop every step of the way :slightly_smiling_face:
 
-    To start, did you have a club meeting this week?
+    Before we get started, I just wanted to let you know that I've started cosplaying as GLaDOS and I've learned a few new tricks (thanks to some awesome suggestions that y'all sent to my creators).
 
-greeting: Hey <%= info[:first_name] %>! Did you have a club meeting this week?
+    Try saying `hackbot gif cave johnson` or `hackbot stats` in a channel that I'm in.
+
+    Let's get started. Did you have a club meeting this week?
+
+greeting: >
+    Hey there!
+
+    Before we get started with check-ins, I just wanted to let you know that I've started cosplaying as GLaDOS and have learned a few new tricks (thanks to some awesome suggestions that y'all sent to my creators).
+
+    Try saying `hackbot gif cave johnson` or `hackbot stats` in a channel that I'm in.
```

Comment:
We should encourage people to tag hackbot properly:

ie. `@hackbot gif cave johnson`